FlowCal/mef.py

```diff
@@ -71,6 +76,19 @@ def clustering_gmm(data,
     events. For more information, consult ``scikit-learn``'s documentation.
 
     """
+    # Check that ``scale`` is supported
+    if scale!='linear' and scale!='log':
+        raise ValueError("scale {} not supported".format(scale))
+
+    # Rescale if necessary
+    if scale=='log':
+        # Copy events
+        data = data.copy()
+        # Log scale requires saturating negative and zero-valued events
+        data[data < 1] = 1.
+        # Rescale
+        data = np.log10(data)*256
```

Comment:
where did the `*256` come from? that seems like an artifact from a previous (less general) version of this. if you want to include that scaling term, seems like this should really be transformed with the inverse of `to_rfi()` which would appropriately incorporate the amplifier resolution/range. that would go back to channel units, though. I think i'm in favor of just a standard natural log transformation on RFI values. (e.g. `np.log(data)`? where `data` should always be in RFI now?)